src/vl-rich-data.js

```diff
@@ -323,7 +340,7 @@ export class VlRichData extends vlElement(HTMLElement) {
         this.__processSearchFilter();
       }
     });
-    observer.observe(this, {childList: true});
+    observer.observe(this, {childList: true, subtree: true});
```

Comment:
Onduidelijk waarom deze code wijziging werd uitgevoerd. Na een revert faalt er geen test. Indien deze code wijziging wel nodig is, kan dit best blijken uit de testen om regressie te voorkomen. Voorlopig werd de code gereverted.

Reply:
Zie PR #40.